bodhi/client/bindings.py

```diff
@@ -287,14 +287,17 @@ def save_override(self, nvr, duration, notes):
         """
         expiration_date = datetime.datetime.utcnow() + \
             datetime.timedelta(days=duration)
-
-        return self.send_request(
-            'overrides/', verb='POST', auth=True, data={
-                'nvr': nvr,
+        data = {'nvr': nvr,
                 'expiration_date': expiration_date,
                 'notes': notes,
                 'csrf_token': self.csrf(),
-            })
+                'duration': duration}
```

Comment:
Hum it does not but we have the `--duration` in the CLI , which is apparently not used.  I guess I will remove this and I can create an issue to implement the server side to set the duration using the CLI.